source/process/product-recipes-template.rst

```diff
@@ -0,0 +1,60 @@
+Product Recipe Template
+========================
+
+This template defines how a product recipe can be added for the depending
```

Comment:
Consider edits to this section "for the depending people" is confusing to me.  I did find this existing documentation as well: https://docs.mattermost.com/process/guest-article.html#mattermost-recipes. We should try to incorporate this information into this template.

Reply:
What do you suggest for https://github.com/mattermost/docs/issues/2718?